functions/source/CopyAitDeployment/copy_ait_deployment.py

```diff
@@ -0,0 +1,152 @@
+    """
+
+    print(json.dumps(event, default=str))
+    responseData = {}
+    status = cfnresponse.FAILED
+    dir_path = "/tmp"
+    BUCKET_NAME = "ast-sandbox-config"
+
+    if event["RequestType"] == "Create":
+        responseData["LambaTest"] = "Create"
+        status = cfnresponse.SUCCESS
+
+        ## AIT Core
+        # Build directory AIT core directory
+        print("Downloading AIT-Core")
+        pathlib.Path("/mnt/efs/ait/AIT-Core/").mkdir(parents=True, exist_ok=True)
+        # Download and place into AIT
+        ait_url = (
+            "https://github.com/NASA-AMMOS/AIT-Core/archive/refs/tags/2.3.5.tar.gz"
+        )
+        download_tar_gz(ait_url, path=dir_path)
+        shutil.copytree(
+            "/tmp/AIT-Core-2.3.5", "/mnt/efs/ait/AIT-Core", dirs_exist_ok=True
+        )
+
+        ## AIT GUI
+        # Build directory AIT GUI directory
+        print("Downloading AIT GUI")
+        pathlib.Path("/mnt/efs/ait/AIT-GUI/").mkdir(parents=True, exist_ok=True)
+        # Download and place into AIT
+        ait_url = "https://github.com/NASA-AMMOS/AIT-GUI/archive/refs/tags/2.3.1.tar.gz"
+        download_tar_gz(ait_url, path=dir_path)
+        shutil.copytree(
+            "/tmp/AIT-GUI-2.3.1", "/mnt/efs/ait/AIT-GUI", dirs_exist_ok=True
+        )
+
+        ## AIT DSN
+        # Build directory AIT DSN directory
+        print("Downloading AIT DSN Plugin")
+        pathlib.Path("/mnt/efs/ait/AIT-DSN/").mkdir(parents=True, exist_ok=True)
+        # Download and place into AIT
+        ait_url = "https://github.com/NASA-AMMOS/AIT-DSN/archive/refs/tags/2.0.0.tar.gz"
+        download_tar_gz(ait_url, path=dir_path)
+        shutil.copytree(
+            "/tmp/AIT-DSN-2.0.0", "/mnt/efs/ait/AIT-DSN", dirs_exist_ok=True
+        )
+
+        # Build necessary folders for the AIT DSN plugin
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/outgoing").mkdir(
+            parents=True, exist_ok=True
+        )
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/incoming").mkdir(
+            parents=True, exist_ok=True
+        )
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/tempfiles").mkdir(
+            parents=True, exist_ok=True
+        )
+        pathlib.Path("/mnt/efs/ait/AIT-Core/ait/dsn/cfdp/datasink/pdusink").mkdir(
+            parents=True, exist_ok=True
+        )
+
+        # Download influx dB
+        print("Downloading Influx DB RPM")
+        download_file(
+            url="https://repos.influxdata.com/rhel/6/amd64/stable/influxdb-1.2.4.x86_64.rpm",
+            filename="/tmp/influxdb-1.2.4.x86_64.rpm",
+        )
+        pathlib.Path("/mnt/efs/ait/setup").mkdir(parents=True, exist_ok=True)
+        shutil.copyfile(
+            "/tmp/influxdb-1.2.4.x86_64.rpm",
+            "/mnt/efs/ait/setup/influxdb-1.2.4.x86_64.rpm",
+        )
+
+        ## Configuration files from S3
+        print("Downloading CONFIG files")
+        s3 = boto3.client("s3")
```

Comment:
Want to use a `boto3.resource(s3)` here as we do above? the high-level client works nicely for simple stuff like this